src/main/java/alfio/config/WebSecurityConfig.java

```diff
@@ -1,48 +1,58 @@
 /**
  * This file is part of alf.io.
- *
+ * <p>
```

Comment:
please remove the `<p>` tags, as they break the "license verifier" build step

Reply:
done